h2o-py/h2o/h2o.py

```diff
@@ -1247,7 +1248,18 @@ def make_metrics(predicted, actual, domain=None, distribution=None):
               data={"domain": domain, "distribution": distribution})
     return res["model_metrics"]
 
+def flow(ip="localhost",port=54321,new = 1):
+    """
+    Open H2O Flow in your browser.
 
+    :param ip: The ip address (or host name) of the server where H2O is running.
+    :param port: Port number that H2O service is listening to.
+    :param browser_type: If browser_type is 0, the url is opened in the same browser window if possible.
```

Comment:
@navdeep-G @mmalohlava I know that this functionality is built into base R so that makes it easy to add it to the R package, but are we proposing to add a new dependency to the **h2o** Python module?  (e.g. webbrowser) Or will people just be able to use it when they happen to have the webbrowser package installed?  How do we document packages that provide optional functionality but are not required (for example, Python pandas)?